_docs/principles/accessibility.md

```diff
@@ -198,61 +301,67 @@ Helix minimum spacing standards require: 
+
+Users that use screen readers or the keyboard to navigate to and select components rely on those components to be visible. Don't hide components as a way to reduce visual clutter. If a control cannot be seen, it cannot be verbally selected.
+
+### Write meaningful alternative text for images
+
+Screen readers read the text you enter in the `<alt>` tag that you associate with an image. If an image provides important contextual information, then use the `<alt>` tag to describe what is happening in the picture. If the image is decorative and does not provide important information, then add an empty `<alt>` tag. If you do not add an `<alt>` tag, some screen readers read the name of the image file, which introduces unnecessary confusion and results in a poor user experience.
 
 {% endcolumn %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+</div>
 
-#### Focus for Input Field
+</section>
 
-In this example, we provide a clear visual feedback that the focus has moved to the input field by changing the border color and adding a highlight to give the user clear indication of where they’re currently taking action.
+<section class="static-section"  markdown="1">
 
-{% endcolumn %}
+<div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% figure [caption:"Default state"][class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-3.svg"/>
-{% endfigure %}
+## Text guidelines
 
-{% endcolumn %}
+Consider the following text guidelines when you construct an accessible UI.
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+### Be succinct
 
-{% figure [caption:"Focused state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-4.svg"/>
-{% endfigure %}
+Write short labels and be concise when writing helper text. Listening to lengthy field names and links can be cumbersome for users that use screen readers. The shorter the label, the more quickly a screen reader can work through the content.
+
+### Identify required fields
+
+Use the HTML5 `required` attribute to distinguish between required and optional fields. Screen readers cannot interpret an asterisk (``*``), which is the convention used to visually identify required fields.
+
+### Link text should convey what happens when clicked
+
+Sentences that include links should include a text description before the link and then a unique name for the link. Do not use **Click Here** as a link naming convention. For example, use 'Visit the [Professional Services](http:www) page to learn more about how we can help transform your IT infrastructure'. Do not use 'Click [here](http:www) to learn more about how we can help transform your IT infrastructure'.
```

Comment:
Should your link URLs be broken?

Reply:
Well, I was trying to mimic what a link looks like in Markdown without actually linking to a URL. Not sure how else to do it.

Reply:
well, I did decide to put a legit URL in there, so we're good! Done.